From 56cdef47103ec39727c735d371a861e6d308f851 Mon Sep 17 00:00:00 2001
From: Tristan de Cacqueray <tdecacqu@redhat.com>
Date: Mon, 1 Jun 2020 02:52:44 +0000
Subject: [PATCH 2/2] Fix boost 1.73.0 support (#4983)

fix boost 1.73.0 support by including the appropriate header and using up-to-date endianness macros
---
 external_libraries/oscpack_build.cpp | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/external_libraries/oscpack_build.cpp b/external_libraries/oscpack_build.cpp
index 3a658f3d3..de7e78fef 100644
--- a/external_libraries/oscpack_build.cpp
+++ b/external_libraries/oscpack_build.cpp
@@ -1,8 +1,8 @@
-#include <boost/detail/endian.hpp>
+#include <boost/predef.h>
 
-#ifdef BOOST_LITTLE_ENDIAN
+#if BOOST_ENDIAN_LITTLE_BYTE
 #define OSC_HOST_LITTLE_ENDIAN
-#elif defined(BOOST_BIG_ENDIAN)
+#elif BOOST_ENDIAN_BIG_BYTE
 #define OSC_HOST_BIG_ENDIAN
 #else
 #error please define endianness
-- 
2.21.3

